lte/gateway/c/core/oai/test/spgw_task/BUILD.bazel

```diff
@@ -72,3 +72,26 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_test(
+    name = "spgw_service_impl",
```

Comment:
Here and below, can these have `size = "small",`?

Reply:
good call! will add